packages/cactus-plugin-ubiquity/package.json

```diff
@@ -0,0 +1,102 @@
+{
+    "name": "@hyperledger/cactus-plugin-ubiquity",
+    "version": "1.0.0",
```

Comment:
@RafaelAPB Please bump this to 1.1.0 and also bump the dependencies versions that are cactus packages (because we have issued a release in the meantime)